script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt

```diff
@@ -28,33 +28,33 @@ We've seen each other a few times when you were a kid, albeit not like this. Eve
 <Page7>
 Saiki...?
 I wonder if there is any branch of the Tohno family with such a name.
-No, before that―――this person is, how should I say―――
+No, before that―――this person is, how do I say it―――
 <Page8>
 "It's alright, I'm used to being looked at strangely.
 I burned my face in an accident. For a long time now, I haven't been able to go outside without wearing them. Sunlight is especially harsh.
-Akiha also dislikes me, but that can't be helped. She is also partly responsible for this."
+Akiha also considers me to be a nuisance but that can't be helped. She is also partly responsible for this after all." // 秋葉にも<煙|けむ>たがられているが、こればかりは仕方のない事だ。彼女にも責任の一端はある事だしね
 <Page9>
-Saiki walks over to the tea wagon, makes himself a cup of tea, and brings it to his lips.
-It is an exaggerated gesture, but it didn't seem out of place.
+Saiki walks over to the trolley, makes himself a cup of tea, and brings it to his lips.
+It is an exaggerated gesture, but it doesn't seem out of place.
 It is a gesture of one born with natural dignity... a casual elegance.
 <Page10>
 "However. My, how dare you return just like that."
 "...Eh?"
 "I'm talking about you, Shiki-kun. You broke Makihisa's orders and escaped from the mansion, and as a result, you were involved in a stupid accident and deserted."
 <Page11>
-"Deserted. Yes, deserted. You who threw away your parent's investment and abandoned your duty, yet here you are, back at the mansion, trying to act like Akiha's brother.
-Don't you feel ashamed of yourself?"
+"Deserted. Yes, deserted. You discarded your parent's investment and abandoned your duty. Yet here you are, back at the mansion, trying to act like Akiha's brother.
+Do you not feel ashamed of yourself?"
 <Page12>
 "――――――"
-I was just about to impulsively stand up but checked myself, before leaning my back deeply against the sofa.
+About to impulsively stand up, I checked myself, leaning back deeply into the sofa.
 There is obvious contempt and provocation in Saiki's eyes, but there is no point in making a scene here. After all, there is no mistake in what he said.
 <Page13>
 "Oh. Don't you have any objections?"
 "No, it's true that I abandoned my duty.
 But don't you think it's shameful for you to use words like 'deserted' and 'investment' when talking about family?"
 <Page14>
-"I chose those words exactly because we were talking about family.
-Fostering is an investment in the family's future. Apart from the city dwellers, individual freedom does not exist in a chosen bloodline or in a noble family line.
+"I chose those words precisely because we were talking about family.
```

Comment:
I honestly prefer "exactly" here but I guess its a matter of style. we should decide on one and introduce it in both files.